api/v1/verticadb_webhook.go

```diff
@@ -211,6 +211,18 @@ func (v *VerticaDB) hasValidInitPolicy(allErrs field.ErrorList) field.ErrorList
 	case CommunalInitPolicyCreate:
 	case CommunalInitPolicyCreateSkipPackageInstall:
 	case CommunalInitPolicyRevive:
+		if v.IsRestoreConfig() && !v.Spec.RestorePoint.IsValidRestorePointPolicy() {
```

Comment:
I suggest you check `RestorePoint` in its own function and there you can just exit earlier if the initPolicy doesn't support it.

Also the error msg is too long.

Reply:
I can move the logic into its own function to make it more clear and readable, though it won't necessarily let us exit earlier as anyway all the checks in the webhook will be executed.